exercises/en/test_05_18.py

```diff
@@ -0,0 +1,13 @@
+def test():
+    # Here we can either check objects created in the solution code, or the
+    # string value of the solution, available as __solution__. A helper for
+    # printing formatted messages is available as __msg__. See the testTemplate
+    # in the meta.json for details.
+
+    # If an assertion fails, the message will be displayed
+    assert "count" in __solution__, "Make sure you are naming your object 'count'."
+    assert "count = 0" in __solution__, "Make sure you are creating the 'count' variable and giving it a value of '0'"
+    assert "for data in dataframes" in __solution__, "Make sure you are interating over 'dataframes'"
```

Comment:
too specific -> cant guarantee they will choose `data` as the element variable name